tests/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+readonly SCRIPT_PATH=$(realpath $(dirname "$0"))
+PROJECT_FOLDER=$(dirname "$SCRIPT_PATH")
+BINARY_FOLDER="$PROJECT_FOLDER/tests/assets"
+INPUT_BIN="$BINARY_FOLDER/nvm3_7_18_1.bin"
+SCHEMA_FILE="$PROJECT_FOLDER/zwave_data_description_scheme.json"
+MIGRATION_TOOL="$PROJECT_FOLDER/build/z-wave-nvm-migration-tool"
+HARDWARE_PART="EFR32XG23"
+TMPDIR=$(mktemp -d)
+status=0
+for VERSION in 7.19.0 7.20.0 7.21.0 7.22.0 7.23.0 7.24.0; do
+  
+  VERSION_=$(echo "$VERSION" | sed 's/\./_/g')
+  OUTPUT_BIN="${TMPDIR}/nvm3_${VERSION_}.bin"
+  EXPECTED_BIN="$BINARY_FOLDER/expected_${VERSION_}.bin"
+
+  echo "Migration from 7.18.1 to $VERSION..."
+  $MIGRATION_TOOL -m "$INPUT_BIN" "$VERSION" $HARDWARE_PART $SCHEMA_FILE -o "$OUTPUT_BIN" > /dev/null 2>&1
```

Comment:
```suggestion
  $MIGRATION_TOOL -m "$INPUT_BIN" "$VERSION" $HARDWARE_PART $SCHEMA_FILE -o "$OUTPUT_BIN" > /dev/null 2>&1 || status=$(1 + $status)
```
To prevent set -e breakage